### PR TITLE
UI Revamp

### PR DESCRIPTION
**Pull request by mikolajmirko**
UI Revamp:
app bar change,
button placement,
page transitions,
visual fixes,
text changes,
statistics mentions removed 

+ UI for friend plans page (logic and more will come in another marge request)

### CODE REVIEW

**Comment by Evil00787** [comment]
Could you please center loading indicator? :)
image

File: lib/logic/child/child_rewards_cubit.dart [hunk 1]
```diff
@@ -65,6 +66,7 @@ class ChildRewardsCubit extends ReloadableCubit {
 				Points.fromUICurrency(UICurrency(type: e.type, title: e.title), e.quantity, creator: e.createdBy)).toList()
 			);
 			await _notificationService.sendRewardBoughtNotification(model.id, model.name, child.connections.first, child);
+			// _authBloc.add(AuthenticationActiveUserUpdated(child.copyWith(points: List.from(child.points), rewards: List.from(child.rewards))));
```
Comment (stasgora) [inline lib/logic/child/child_rewards_cubit.dart:69]: Is that the only place? What about `CaregiverAwardsCubit`?

**Review by stasgora — CHANGES_REQUESTED** [review]
Great redesign :heart:

File: i18n/en_US.json [hunk 2]
```diff
@@ -119,13 +118,14 @@
...
-					"pageHint": "Manage child profiles and connected caregiver accounts"
+					"rateTasks": "Rate Tasks"
 				},
 				"content": {
 					"childProfilesTitle": "Child profiles",
+					"childProfilesSubtitle": "Children earn points by doing tasks",
 					"noChildProfilesAdded": "No children assigned",
-					"caregiverProfilesTitle": "Befriended caregivers",
+					"caregiverProfilesTitle": "Friends",
+					"caregiverProfilesSubtitle": "Befriended caregivers and their plans",
```
[the remark this answers is not in the document]
Comment (mikolajmirko) [reply i18n/en_US.json:128]: I'm open to suggestions. I wanted to write sub title everywhere, but sometimes they sound forced :c

File: lib/logic/child/child_rewards_cubit.dart (hunk 1 above)
Comment (mikolajmirko) [reply lib/logic/child/child_rewards_cubit.dart:69]: Oh yeah, adding it there didn't change anything, so I commented this out and then forgot, ops! Should I put it back?

**Comment by mikolajmirko** [comment]
> Could you please center loading indicator? :)
> image

I see a old code there, you sure it's wrong on the new design? I just checked it out and it seems centred everywhere. We now use appbar from scaffold, so there is no need for a column with bar and content like before. `wrapWithExpanded `is used only in Columns.

obraz

File: lib/logic/child/child_rewards_cubit.dart (hunk 1 above)
Comment (stasgora) [reply lib/logic/child/child_rewards_cubit.dart:69]: I though we had a problem with points not updating here.. is it resolved without this call?

Comment (mikolajmirko) [reply lib/logic/child/child_rewards_cubit.dart:69]: The `_authBloc` didn't help and I reverted to updating the widget through its constructor.

Comment (stasgora) [reply lib/logic/child/child_rewards_cubit.dart:69]: Ok then, if it works there is probably no point in keeping this here

File: i18n/en_US.json (hunk 2 above)
[the remark this answers is not in the document]
Comment (stasgora) [reply i18n/en_US.json:128]: Is **List of befriended caregivers** to obvious?

[the remark this answers is not in the document]
Comment (mikolajmirko) [reply i18n/en_US.json:128]: It's not bad. It says what friends are in short sentence 👍

File: i18n/pl_PL.json [hunk 3]
```diff
@@ -119,13 +118,14 @@
...
-					"pageHint": "Zarządzaj profilami dzieci i powiązanymi kontami opiekunów"
+					"rateTasks": "Oceń zadania"
 				},
 				"content": {
 					"childProfilesTitle": "Profile dzieci",
+					"childProfilesSubtitle": "Dzieci zdobywają punkty za wykonywanie zadań",
 					"noChildProfilesAdded": "Brak przypisanych dzieci",
-					"caregiverProfilesTitle": "Zaprzyjaźnieni opiekunowie",
+					"caregiverProfilesTitle": "Znajomi",
+					"caregiverProfilesSubtitle": "Zaprzyjaźnieni opiekunowie i ich plany",
```
Comment (stasgora) [inline i18n/pl_PL.json:128]:
```suggestion
					"caregiverProfilesSubtitle": "Lista zaprzyjaźnionych opiekunów",
```

**Review by stasgora — APPROVED** [review]

**Pull request merged**